ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -115,43 +121,64 @@ codeAction state _ (CodeActionParams _ _ (TextDocumentIdentifier uri) _range Cod
+  & defCodeAction #removeRedundantImport                       "Remove redundant imports"                                 ([] :: [(T.Text, TextEdit)])
+  & defCodeAction #removeInvalidExport                         "Remove invalid exports"                                   ([] :: [(T.Text, TextEdit)])
+  & defCodeAction #suggestSignature                            "Add type signature for bindings"                          (suggestSignature True)
+  & defCodeAction #suggestExtendImport                         "Extend import lists"                                      suggestExtendImport
+  & defCodeAction #suggestImportDisambiguation                 "Disambiguate imports"                                     suggestImportDisambiguation
+  & defCodeAction #importClassMethod                           "Add new imports or extend import lists for class methods" suggestNewOrExtendImportForClassMethod
+  & defCodeAction #suggestFillTypeWildcard                     "Fill wildcards in types"                                  suggestFillTypeWildcard
+  & defCodeAction #suggestFixConstructorImport                 "Fix data constructor imports"                             suggestFixConstructorImport
+  & defCodeAction #suggestModuleTypo                           "Fix import module names"                                  suggestModuleTypo
+  & defCodeAction #suggestReplaceIdentifier                    "Replace identifiers"                                      suggestReplaceIdentifier
+  & defCodeAction #removeRedundantConstraints                  "Remove redundant constraints from type signatures"        removeRedundantConstraints
+  & defCodeAction #suggestAddTypeAnnotationToSatisfyContraints "Add type annotations"                                     suggestAddTypeAnnotationToSatisfyContraints
+  & defCodeAction #suggestConstraint                           "Add type constraints"                                     suggestConstraint
+  & defCodeAction #suggestImplicitParameter                    "Add implicit parameters"                                  suggestImplicitParameter
+  & defCodeAction #suggestHideShadow                           "Hide shadowed identifiers from imports"                   suggestHideShadow
+  & defCodeAction #suggestNewDefinition                        "Add new definitions to be implemented"                    suggestNewDefinition
+  & defCodeAction #suggestNewImport                            "Add new imports"                                          suggestNewImport
+  & defCodeAction #suggestDeleteUnusedBinding                  "Remove unused bindings"                                   suggestDeleteUnusedBinding
+  & defCodeAction #suggestExportUnusedTopBinding               "Export unused bindings"                                   suggestExportUnusedTopBinding
+  & defCodeAction #suggestFillHole                             "Fill holes with suggestions from GHC"                     suggestFillHole -- Lowest priority
+
```

Comment:
It would be great if you add some tests for these configs
(like these actually disable some actions/etc.)